### PR TITLE
OSDOCS-2088: Added a Quick Start highlighting markdown reference

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-2088
Preview build: https://deploy-preview-33014--osdocs.netlify.app/openshift-enterprise/latest/web_console/creating-quick-start-tutorials?utm_source=github&utm_campaign=bot_dp#quick-start-highlighting-reference_creating-quick-start-tutorials

### CODE REVIEW

**Review by sahil143 — COMMENTED** [review]
Looks good overall

**Review by gdoyle1 — APPROVED** [review]
Nice looks good!

File: modules/quick-starts-highlighting-reference.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+[id="quick-start-highlighting-reference_{context}"]
+= Quick start highlighting markdown reference
+
+[id="quick-start-highlighting-perspective-switcher_{context}"]
+== Perspective switcher
+
+[source,yaml]
+----
+[link]{{highlight qs-perspective-switcher}}
```
[the remark this answers is not in the document]
Comment (ahardin-rh) [reply modules/quick-starts-highlighting-reference.adoc]: @sahil143 Updated. Thanks!

**Comment by ahardin-rh** [comment]
@yapei Please provide QE review. Thanks!

File: modules/quick-starts-highlighting-reference.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+[id="quick-start-highlighting-reference_{context}"]
+= Quick start highlighting markdown reference
+
+The highlighting, or hint, feature enables Quick Starts to contain a link that can highlight and animate a component of the web console.
+
+The markdown syntax contains:
+
+* bracketed link text
+* the `highlight` keyword, followed by
+* the ID of the element that you want to animate
```
Comment (bobfuru) [inline modules/quick-starts-highlighting-reference.adoc:14]: I've generally seen bulleted lists capitalize the first letter of each item. ISG check seems to agree though I can't find a rule about it from a quick search. 🤷

Comment (bobfuru) [inline modules/quick-starts-highlighting-reference.adoc:14]: I wouldn't trail between second and third bullets here, but would instead just combine them into one bullet:
"The `highlight` keyword, followed by the ID of the element that you want to animate"

**Review by bobfuru — COMMENTED** [review]
Couple of tiny nits about the bulleted list but in general LGTM! 😁

**Comment by yapei** [comment]
Hi @ahardin-rh , it looks like a feature added by Developer Console team I think we need an ack from Dev Console QE

**Review by sahil143 — APPROVED** [review]

**Comment by ahardin-rh** [comment]
@ppitonak Can you please provide a review or point me in the right direction? Thanks!

**Comment by ahardin-rh** [comment]
@yapei Can you please help me identify a correct Dev Console QE contact? Thanks!

**Comment by yapei** [comment]
> @yapei Can you please help me identify a correct Dev Console QE contact? Thanks!

In Bugzilla I see default `QA Contact` for `Dev Console` is @gajanan-more , could you help review?

**Comment by gajanan-more** [comment]
> > @yapei Can you please help me identify a correct Dev Console QE contact? Thanks!
> 
> In Bugzilla I see default `QA Contact` for `Dev Console` is @gajanan-more , could you help review? Please help redirect to correct contact if I'm wrong

I will have a look @yapei @ahardin-rh

**Review by gajanan-more — APPROVED** [review]
Looks good to me!!

**Comment by ahardin-rh** [comment]
@gajanan-more Thank you for your help!

**Pull request merged**